pom.xml

```diff
@@ -851,6 +852,11 @@
                 <version>1.18.22</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
```

Comment:
Could you put it closer to the `tika` dependencies with some comment that if tika is upgraded we would need to check if hardcoded jdom2 dependency version is still needed?